lib/build/command.js

```diff
@@ -4,6 +4,7 @@ var common = require('../common/command');
 var isChildProcess = typeof process.send == 'function'; // child process has send method
 var targets = ['fs', 'output-graph', 'none']; // first is default
 var jsBundleFormats = ['js', 'json'];
+var DEFAULT_PACK_CONFIG = { compress: { ecma: 5 }, output: { ecma: 5 } };
```

Comment:
DEFAULT_PACK_CONFIG should be undefined by default. Since every single minifier can has its own options